news/2019-05-27-osutaiko-world-cup-2019-concludes.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: "osu!taiko World Cup 2019 Concludes"
+date: 2019-05-27 22:00:00 +0000
+---
+ 
+*sniff sniff...* do you smell that? Yeah, smelt like ...*sniff*... poppin' gamers in here. What's poppin' gamers? The osu!taiko World Cup just came to a close last weekend - read on to find out what happened in the Grand Finale!
+ 
+![](/wiki/shared/news/banners/TWC_2019_banner.jpg)
+ 
+Six weeks ago, the osu!taiko World Cup stepped boldly into the limelight, returning for its 9th instalment. 
+
+We've come a long way since our inception in 2011 — long gone are the days where multiple country teams run rampant, with dominating performances from the likes of both Japan Team A and B, and curious oddities such as the great Italy-Netherlands Alliance of 2011-2013.
```

Comment:
```suggestion
We've come a long way since our inception in 2011—long gone are the days where multiple country teams run rampant, with dominating performances from the likes of both Japan Team A and B, and curious oddities such as the great Italy-Netherlands Alliance of 2011-2013.
```